pkg/scheduler/api/types.go

```diff
@@ -335,6 +336,15 @@ type HostPriority struct {
 	Score int
 }
 
+// FeatureDependency provides the feature and its dependent predicates and priorities.
+type FeatureDependency struct {
+	Name                  string      // Name of the feature gate.
+	NeededPredicateList   sets.String // List of predicates needed if we enable this predicate or priority
+	NeededPriorityList    sets.String // List of priorities needed if we enable this predicate or priority.
+	ExcludedPredicateList sets.String // List of predicates that should be excluded if we enable this predicate/priority.
+	ExcludedPriorityList  sets.String // List of priorities that should be excluded if we enable this predicate/priority
```

Comment:
The convention is to have the comment above the field, not on the same line.

Reply:
fixed